distribution/resources/bin/gateway

```diff
@@ -201,14 +293,13 @@ EOF
   done
 elif [ "$CMD" = "stop" ]; then
   export GWHOME="$GWHOME"
-  kill -term `cat "$GWHOME"/bin/gateway.pid`
-  kill -9 `lsof -t $GWHOME/logs/access_logs`
+  kill -term `cat $GWHOME/bin/gateway.pid`
   rm -f file $GWHOME/bin/gateway.pid
+  echo "Microgateway has been stopped successfully"
```

Comment:
Shall we check `$? = 0` before saying success?

Reply:
fixed with https://github.com/wso2/product-microgateway/pull/944/commits/94a724d3ba8f79d677f4d2d3ec1e94125ec01db1